src/lib/holocene/chip.svelte

```diff
@@ -30,7 +30,7 @@
 
 <style lang="postcss">
   .chip {
-    @apply flex h-8 w-fit min-w-fit flex-row items-center justify-between gap-1 whitespace-nowrap break-all rounded-md border border-subtle bg-slate-50 p-1 text-sm text-slate-700;
+    @apply surface-subtle flex h-8 w-fit min-w-fit flex-row items-center justify-between gap-1 whitespace-nowrap break-all rounded-md border border-subtle p-1 text-sm text-primary;
```

Comment:
I think surface-subtle already has text-primary, so you should be able to remove the extra one here